types/react-big-calendar/react-big-calendar-tests.tsx

```diff
@@ -104,10 +104,12 @@ class CalendarResource {
 
 // Drag and Drop Example Test
 {
+    class MyCalendar extends Calendar<CalendarEvent, CalendarResource> {}
+
     interface Props {
         localizer: DateLocalizer;
     }
-    const DragAndDropCalendar = withDragAndDrop(Calendar);
+    const DragAndDropCalendar = withDragAndDrop<CalendarEvent, CalendarResource>(MyCalendar);
```

Comment:
Do we need to explicitly pass generic arguments?

Reply:
I couldn't figure out how to define this stuff with type safety *without* explicitly passing them (or type casting, which I found worse)

Reply:
> I couldn't figure out how to define this stuff with type safety

What do you mean by this?

Reply:
`withDragAndDrop` was not passing the generic parameters of `TEvent` and `TResource` (old code: `declare function withDragAndDrop(calendar: typeof Calendar): typeof DragAndDropCalendar;`). My change adds this, but it found no way how to achieve this which TypeScript inferring them, so it have to pass them explicitly via `const DragAndDropCalendar = withDragAndDrop<CalendarEvent, CalendarResource>(MyCalendar)`.